opm/parser/eclipse/EclipseState/Aquifer/NumericalAquifer/NumericalAquifers.hpp

```diff
@@ -46,10 +46,14 @@ namespace Opm {
 
         std::unordered_map<size_t, double> aquiferCellVolumes() const;
 
-        std::vector<NNCdata> aquiferNNCs(const EclipseGrid& grid, const FieldPropsManager& fp) const;
+        std::vector<NNCdata> aquiferCellNNCs() const;
+        std::vector<NNCdata> aquiferConnectionNNCs(const EclipseGrid& grid, const FieldPropsManager& fp) const;
 
         std::unordered_map<size_t, AquiferCellProps> aquiferCellProps() const;
 
+        void preGeneratingConnections(const Deck& deck, const EclipseGrid& grid);
```

Comment:
These functions should not have the `xxxing` form in the name - rather:
```
void preGenerateConnections(); // or initConnections()?
void postProcessConnections();
```
since this is so complex it is worth it to fiddle with the names - sorry.